zmon_worker_monitor/web_server/web.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Logic for a basic HTTP server (flask app). This code will most likely be run in a separate process.
+We expose several HTTP endpoints that return information about the operations of zmon-worker processes.
+
+This app obtain information on the operation of zmon-worker by querying ZMON's internal RPC server.
+Said RPC server exposes the methods of ProcessController, our process supervisor, which lives in the
+parent process.
+
+"""
+
+import logging
+import json
+from flask import Flask, Response, request, url_for, redirect, jsonify
+
+from ..rpc_client import get_rpc_client_plus
+
+
+app = Flask(__name__)
+
+
+_rpc_endpoint = 'http://localhost:8000/rpc_path'
+
+
+_rpc_client = None
+
+
+def get_logger():
+    return logging.getLogger(__name__)
+
+
+@app.route('/')
+def root():
+    return 'Hello World!'
+
+
+@app.route('/api/running_procs/')
```

Comment:
Yes, although I was thinking in having two subtrees /api/ and /html/
Under /html/ we could put some pages for humans... maybe just one index page with a quick view of the state of the system, performance graphics, and problems?